src/main/java/it/gov/pagopa/receipt/pdf/notifier/Info.java

```diff
@@ -0,0 +1,37 @@
+package it.gov.pagopa.receipt.pdf.notifier;
+
+import com.microsoft.azure.functions.ExecutionContext;
+import com.microsoft.azure.functions.HttpMethod;
+import com.microsoft.azure.functions.HttpRequestMessage;
+import com.microsoft.azure.functions.HttpResponseMessage;
+import com.microsoft.azure.functions.HttpStatus;
+import com.microsoft.azure.functions.annotation.AuthorizationLevel;
+import com.microsoft.azure.functions.annotation.FunctionName;
+import com.microsoft.azure.functions.annotation.HttpTrigger;
+
+import java.util.Optional;
+
+
+/**
+ * Azure Functions with Azure Http trigger.
+ */
+public class Info {
+
+    /**
+     * This function will be invoked when a Http Trigger occurs
+     *
+     * @return response with HttpStatus.OK
+     */
+    @FunctionName("Info")
```

Comment:
as reminder we can expose all info service on apim receipt status product, to our [internal status pages](https://pagopa.github.io/pagopa-status-page/#/)

Reply:
Moving comment to PR https://github.com/pagopa/pagopa-receipt-pdf-notifier/pull/5